src/Executables/CombineH5/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(EXECUTABLE CombineH5)
+
+add_spectre_executable(
+  ${EXECUTABLE}
+  EXCLUDE_FROM_ALL
+  CombineH5.cpp
+  )
+
+target_link_libraries(
+  ${EXECUTABLE}
+  PRIVATE
+  Boost::boost
+  Boost::program_options
+  DataStructures
+  IO
+  Parallel # Only for printf
```

Comment:
I don't think we've been documenting what parts of the libs we've used elsewhere in our CMake, so I'd remove this comment